### PR TITLE
Issues resolver

### PR DESCRIPTION
**Pull request by jitendrajaria**
pivotal issues 
1) address line unique constraint removed
2) Inquiry address entity updated
3)Inquiry status default value set to OPEN
are solved .

### CODE REVIEW

File: src/main/java/creativei/entity/Locality.java [hunk 1]
```diff
@@ -14,6 +16,13 @@
     @ManyToOne
     private City city;
 
+    public Locality(){}
+
+    public Locality(AddressVo addressVo){
```
Comment (aayushKhandpur) [inline src/main/java/creativei/entity/Locality.java]: Why do you need the whole addressVo here? Pass multiple params like (locationId, area, pincode)

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address comments

File: src/main/java/creativei/entity/Locality.java (hunk 1 above)
Comment (jitendrajaria) [reply src/main/java/creativei/entity/Locality.java]: okay Issue resolved in commit  4e98a52

Comment (jitendrajaria) [reply src/main/java/creativei/entity/Locality.java]: more accurately constructor is present in this commit f4529da

File: src/main/java/creativei/entity/InquiryAddress.java [hunk 2]
```diff
@@ -21,7 +21,7 @@ public InquiryAddress(AddressVo addressVo){
         this.addressLine1=addressVo.getAddressLine1()==null||addressVo.getAddressLine1().isEmpty()?null:addressVo.getAddressLine1();
         this.country=addressVo.getCountry()==null||addressVo.getCountry().isEmpty()?null:addressVo.getCountry();
         this.state=addressVo.getState()==null||addressVo.getState().isEmpty()?null:addressVo.getState();
-        this.location=new Locality(addressVo.getLocationId(),addressVo.getArea(),addressVo.getPin());
+        this.location=new Locality(addressVo.getLocationId());
```
Comment (aayushKhandpur) [inline src/main/java/creativei/entity/InquiryAddress.java]: How are we setting pincode and area?

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address comment

File: src/main/java/creativei/entity/InquiryAddress.java (hunk 2 above)
Comment (jitendrajaria) [reply src/main/java/creativei/entity/InquiryAddress.java]: why we need to locality pin code and area when it is already present in our database?

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**